<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KD_Tree_GUIClass</class>
 <widget class="QMainWindow" name="KD_Tree_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KD_Tree_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>684</width>
      <height>684</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ranGenButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Generate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNumber">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearInputButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Input</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputWindow">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>320</y>
      <width>291</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showPointsButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Points</string>
    </property>
   </widget>
   <widget class="QSlider" name="pointSizeChanger">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>730</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="compPerformButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare Performance</string>
    </property>
   </widget>
   <widget class="QSlider" name="compDivChanger">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>100</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QGraphicsView</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="kd_tree_gui.qrc"/>
 </resources>
 <connections/>
</ui>
